pkg/orchestrator/assetscanestimationwatcher/watcher.go

```diff
@@ -237,9 +284,20 @@ func (w *Watcher) reconcilePending(ctx context.Context, assetScanEstimation *mod
 
 	assetScanEstimation.State.LastTransitionTime = utils.PointerTo(time.Now())
 
+	// Set default ttl if not set.
+	if assetScanEstimation.TTLSecondsAfterFinished == nil {
+		assetScanEstimation.TTLSecondsAfterFinished = utils.PointerTo(DefaultAssetScanEstimationTTLSeconds)
+	}
```

Comment:
I think we should probably rename reconcileNoState to reconcileDefaults and always run it before the rest of the reconciling logic. This and the state default can be processed there.